src/page-modules/assistant/layout.tsx

```diff
@@ -54,6 +60,14 @@ function AssistantLayout({ children, tripQuery }: AssistantLayoutProps) {
     getTransportModeFilter,
   );
 
+  useEffect(() => {
+    setDefaultFilters(
+      transportModeFilter
+        ?.filter((filter) => filter.icon.transportMode !== 'air')
```

Comment:
Will this always filter out air, regardless if the user has manually enabled it to see flights?

Reply:
No, the user can manually set check the air filter after the initial search.

Reply:
Right, because the `transportModeFilter` never changes, which means that the `useEffect` only runs once. In theory at least.

Reply:
It sounds like the URL state is not prioritized and that this would override it 🤔

Reply:
My idea was that something like [this](https://github.com/AtB-AS/planner-web/pull/243) would work.

Reply:
Scratch that, everything didn't work.

I think the approach with using `onTransportFilterChanged` might be a good solution.

Also need to make a change with "All", something like below. Setting the filter to all of the options instead of `null`. (Need to handle when `data` is undefined or null or whatever).

https://github.com/AtB-AS/planner-web/blob/2dec3a0de733d707520f6b406dfd81ce2d0f7c2d/src/modules/transport-mode/filter/filter.tsx#L53-L55

```ts
            onChange={(event) => {
              onChangeWrapper(
                event.target.checked ? data?.map((option) => option.id) : [],
              );
            }}
```

Reply:
Okey👍